### PR TITLE
Include ros-vizbox in robocup-at-home

### PR DESCRIPTION
**Pull request by LoyVanBeek**
That is a dependency of tue-dev so you should get the vizbox if all is correct

### CODE REVIEW

File: installer/targets/ros-python-pillow/install.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+- type: pip
+  name: pillow
```
Comment (MatthijsBurgh) [inline installer/targets/ros-python-pillow/install.yaml]: Is there also a debian target?

File: installer/targets/ros-python-tornado/install.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+- type: pip
+  name: tornado
```
Comment (MatthijsBurgh) [inline installer/targets/ros-python-tornado/install.yaml]: Is there also a debian target?

**Comment by MatthijsBurgh** [comment]
@LoyVanBeek do you aprrove my changes? Than I will merge it.

**Comment by LoyVanBeek** [comment]
I just merged your PR in the vibox, so this is good to go as welll

**Pull request merged**